src/bp/nlu-core/intents/oos-intent-classfier.ts

```diff
@@ -268,7 +274,7 @@ export class OOSIntentClassifier implements NoneableIntentClassifier {
     // TODO: proceed to election between none intent and oos, remove none intent and make sure confidences sum to 1.
 
     return {
-      ...intentPredictions,
+      intents: intentPredictions,
```

Comment:
Just to make sure this is intended ? prior to this PR it wasn't nested under intent key. I imagine you changed your typings accordingly.

I'm sorry I don't mean to question you or your decisions. I just want to make sure things didn't slip away while refactoring

Reply:
No this is wanted. 

I wanted to make a unit test to check that this logic works as expected, but I needed to make a fake or mock for `Tools`.

I will add it in an upcoming PR.